framework/packages/abstract-std/src/core/manager.rs

```diff
@@ -47,9 +49,11 @@ pub mod state {
         pub const ACCOUNT_MODULES: &str = "d";
         pub const DEPENDENTS: &str = "e";
         pub const SUB_ACCOUNTS: &str = "f";
+        pub const STATE: &str = "g";
     }
 
     /// Suspension status
+    // TODO: Pull it inside Config as `suspended: Option<String>`, with reason of suspension inside a string?
     pub const SUSPENSION_STATUS: Item<SuspensionStatus> = Item::new(namespace::SUSPENSION_STATUS);
```

Comment:
@CyberHoward What do you think?

Reply:
Think bool is fine for now. We'll want to sync on this functionality with XION to see if they even want it.

Reply:
> Think bool is fine for now. We'll want to sync on this functionality with XION to see if they even want it.

It can be `Option<bool>` as well, I just don't get why we want to have it in the state when most of the time account won't be suspended

Reply:
*inside Config if option

Reply:
True, we can pull it out into an optional that we don't save if not set!